.circleci/config.yml

```diff
@@ -66,9 +66,38 @@ jobs:
       - checkout
       - check-changed:
           patterns: op-bindings,op-chain-ops,packages/
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - yarn-packages-v2-{{ checksum "yarn.lock" }}
       - run:
           name: Install dependencies
-          command: yarn && git diff --exit-code
+          command: yarn install --imutable
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: yarn-packages-v2-{{ checksum "yarn.lock" }}
```

Comment:
I assume this might also be an issue with the yarn version we use.

Reply:
why would the cache be an issue?

Reply:
TBH IDK what this is actually doing, it's just the `v2` that's triggering me.